sootty/storage/wiregroup.py

```diff
@@ -40,9 +38,27 @@ def find(self, name: str):
 
     def get_names(self):
```

Comment:
This change to how wire names are computed should be a new PR, assuming it's separate from the breakpoint logic.

Reply:
now resolved in: https://github.com/Ben1152000/sootty/pull/64